src/prefect/utilities/processutils.py

```diff
@@ -13,6 +26,161 @@
 TextSink = Union[anyio.AsyncFile, TextIO, TextSendStream]
 
 
+if sys.platform == "win32":
+    from ctypes import WINFUNCTYPE, c_int, c_uint, windll
+
+    _windows_process_group_pids = set()
```

Comment:
I dislike this approach, but haven't found a better one if we want to be able to handle CTRL-C presses after we've started Windows processes in new process groups. 

We could use a Python SIGINT handler instead, but that could get replaced elsewhere in the codebase. In contrast, adding a new Win32 console control handler doesn't remove existing handlers.

 `pywin32` offers a more Pythonic ways of working with the win32 API, but it's a large dependency I don't think we want to add.